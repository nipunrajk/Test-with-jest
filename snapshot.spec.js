const user = {
  name: "Nipun Raj k",
  age: "25",
  job: "Developer",
};

test("user matches", () => {
  expect(user).toMatchSnapshot();
});
