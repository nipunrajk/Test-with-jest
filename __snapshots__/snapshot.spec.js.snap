// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user matches 1`] = `
{
  "age": "25",
  "job": "Developer",
  "name": "Nipun Raj k",
}
`;
